### DISCUSSION TITLE
How does xrdp work with multiple users logged in at the same time, without affecting each other?

### DISCUSSION THREAD

**Discussion by MagicStarTrace** [neutrinolabs/xrdp#2206] [Q&A]
server version: ubuntu 18.04

server: xrdp、xorg  

How can we work together so that we can log on to one server at a time without affecting each other?

Thank You!

**Comment by matt335672** [comment]
This is all standard POSIX stuff, and really nothing to do with xrdp. You might get some clues from #2117.

The best thing to do is set up some different users on your Ubuntu machine, and log in with several ssh sessions. You can then play with commands like `id` to get an idea of how [UIDs and GIDs work](https://www.redhat.com/sysadmin/user-account-gid-uid). Following any one of the several UNIX/Linux tutorials on the Internet will also help. Then you've got it - xrdp doesn't use anything special to achieve the same effect.
